.github/workflows/rust.yml

```diff
@@ -61,17 +61,22 @@ jobs:
         run : |
           openssl genrsa -out key.pem 2048
           mv key.pem resources
+
       - name: Run tests
+        env:
+          DATABASE_URL: "postgres://postgres:postgres@localhost/weather_bot"
+          RUST_TELEGRAM_BOT_TOKEN : ${{ secrets.RUST_TELEGRAM_BOT_TOKEN }}
+          OPEN_WEATHER_MAP_API_TOKEN: ${{ secrets.OPEN_WEATHER_MAP_API_TOKEN }}
         uses: actions-rs/cargo@v1
         with:
           command: test
           args: --verbose --all-features
-
-      - name: Run dirty tests
-        uses: actions-rs/cargo@v1
-        with:
-          command: test
-          args: --verbose --all-features -- --ignored
+        
```

Comment:
I think you can remove it if you don't need it

Reply:
Done in [Update rust.yml](https://github.com/pxp9/weather_bot_rust/pull/8/commits/07eb6eb16924afa44b2a762b727bb09f08d9342c)